### PR TITLE
Add ms to duration units

### PR DESCRIPTION
**Pull request by dshokouhi**
## Proposed change

may have been an oversight or miss https://github.com/home-assistant/core/blob/dev/homeassistant/components/sensor/const.py#L177

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Expanded the "duration" device class to include milliseconds (ms) for more precise time interval representation in sensor integrations.

- **Documentation**
	- Updated the "Sensor" integration document to reflect the new unit of measurement while preserving existing content and structure.

### CODE REVIEW

File: source/_integrations/sensor.markdown [hunk 1]
```diff
@@ -59,7 +59,7 @@ The following device classes are supported for sensors:
 - **data_size**: Data size in bit, kbit, Mbit, Gbit, B, kB, MB, GB, TB, PB, EB, ZB, YB, KiB, MiB, GiB, TiB, PiB, EiB, ZiB or YiB
 - **date**: Date string (ISO 8601)
 - **distance**: Generic distance in km, m, cm, mm, mi, nmi, yd, or in
-- **duration**: Duration in d, h, min, or s
+- **duration**: Duration in d, h, min, ms or s
```
Comment (c0ffeeca7) [inline source/_integrations/sensor.markdown]:
```suggestion
- **duration**: Duration in d, h, min, s, or ms
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you for the catch, @dshokouhi :+1:

**Comment by c0ffeeca7** [comment]
Thank you also for the link to the related line core. Makes things much quicker for review. Much appreciated.

**Comment by dshokouhi** [comment]
happy to help :)

**Pull request merged**